base/src/main/java/com/smartdevicelink/proxy/rpc/VideoStreamingCapability.java

```diff
@@ -180,4 +184,14 @@ public VideoStreamingCapability setScale(Double scale) {
         setValue(KEY_SCALE, scale);
         return this;
     }
+
+    @SuppressWarnings("unchecked")
+    public List<VideoStreamingCapability> getAdditionalVideoStreamingCapabilities(){
+        return (List<VideoStreamingCapability>) getObject(VideoStreamingCapability.class, KEY_ADDITIONAL_VIDEO_STREAMING_CAPABILITIES);
+    }
+
```

Comment:
@kostyaBoss above line 193 please include (from the code generator tool):

"    /**
     * Sets the additionalVideoStreamingCapabilities.
     *
     * @param additionalVideoStreamingCapabilities
     * {"array_min_size": 1, "array_max_size": 100}
     * @since SmartDeviceLink 7.0.0
     */"